Tests/Data/Elliptic/circle_radius_1/circle_1e1_axi.prj

```diff
@@ -5,7 +5,7 @@
     <processes>
         <process>
             <name>GW23</name>
```

Comment:
How about change it for a meaningful name?

Reply:
@wenqing This is now 4 years to late for this review ;)
But yes, it could be changed too.

Reply:
When we have agreed on a name I can change the project files, too.